addons/hr_homeworking/static/src/calendar/hr_homeworking_calendar_controller.js

```diff
@@ -0,0 +1,78 @@
+/** @odoo-module **/
+
+import { patch } from "@web/core/utils/patch";
+import { _lt, _t } from "@web/core/l10n/translation";
+import { useService } from "@web/core/utils/hooks";
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { AttendeeCalendarController } from "@calendar/views/attendee_calendar/attendee_calendar_controller"
+import { FormViewDialog } from "@web/views/view_dialogs/form_view_dialog";
+import {
+    serializeDate,
+} from "@web/core/l10n/dates";
+patch(AttendeeCalendarController.prototype, "hr_homeworking_calendar_controller",{
+    setup() {
+        this._super()
+        this.action = useService("action");
+    },
+    async editRecord(record, context = {}, shouldFetchFormViewId = true) {
+        if ('date' in record.rawRecord) {
```

Comment:
I think it's why i check if the date exist in record.rawRecord because it's only exist in homeworking model and not in calendar.event model . But' you're right it's strange i think i will change the key to check :)